osspi/tasks/osspi/run-osspi-docker.yaml

```diff
@@ -0,0 +1,24 @@
+---
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: harbor-repo.vmware.com/source_insight_tooling/osspi-runner
+
+inputs:
+  - name: ci_repo
+params:
+  IMAGE:
+  TAG: latest
+  API_KEY:
+  OSM_ENVIRONMENT: beta
```

Comment:
Should we remove beta here and other places?

Reply:
Let's make production the default!!!